app/code/Magento/GoogleAnalytics/Model/Config/Source/DefaultYes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Google AdWords conversation value type source
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\GoogleAnalytics\Model\Config\Source;
+
+/**
+ * @api
+ * @since 100.0.2
+ */
+class DefaultYes implements \Magento\Framework\Option\ArrayInterface
+{
+    /**
+     * Get default yes option
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        return [
```

Comment:
What's the point of having a select field with only one possible value to select?